### PR TITLE
updated files

### PR DESCRIPTION
**Pull request by amoramicae**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/letum.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
       "email": ""
   },
   "record": {
-    "CNAME": "letum.pages.dev"    
+    "CNAME": "letum.pages.dev",
+    "MX": [
+      "mx1.improvmx.com.",
+      "mx2.improvmx.com."
+    ],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
```
Comment (DEV-DIBSTER) [inline domains/letum.json:13]: Cannot have CNAME record in conjunction with other records.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/letum.json (hunk 1 above)
Comment (amoramicae) [reply domains/letum.json]: can you look into it again? i fixed the issue.

File: domains/mail.letum.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   "owner": {
+      "username": "amoramicae",
+      "discord": "1252631745514180790",
+      "email": ""
+  },
+  "record": {
+    "MX": [
+      "mx1.improvmx.com.",
+      "mx2.improvmx.com."
+    ],
```
Comment (Stef-00012) [inline domains/mail.letum.json:11]: remove the trailing dot
```suggestion
    "MX": [
      "mx1.improvmx.com",
      "mx2.improvmx.com"
    ],
```

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/mail.letum.json (hunk 2 above)
Comment (amoramicae) [reply domains/mail.letum.json:11]: oookkkayyyy ... i just copy pasted the records from improvmx

**Review by Stef-DP — APPROVED** [review]

**Pull request merged**